lib/src/widget/back_button.dart

```diff
@@ -6,36 +8,28 @@ class NeumorphicBackButton extends StatelessWidget {
   final VoidCallback onPressed;
   final NeumorphicStyle style;
   final EdgeInsets padding;
-  final bool reversedArrow;
+  final bool reversedIcon;
 
   const NeumorphicBackButton({
     Key key,
     this.onPressed,
     this.style,
     this.padding,
-    this.reversedArrow = false,
+    this.reversedIcon = false,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    final theme = Theme.of(context);
+    final nThemeIcons = NeumorphicTheme.of(context).current.appBarTheme.icons;
     return NeumorphicButton(
       style: style,
       padding: padding,
       tooltip: MaterialLocalizations.of(context).backButtonTooltip,
-      child: Icon(_getIcon(theme)),
+      child: reversedIcon
+          ? Transform.rotate(
```

Comment:
a bit overkill to introduce transformation for that imho.
Why not adding a `forward` icon instead into your `NeumorphicAppBarIcons` the idea was to have back and forward if needed

Reply:
Ok, I will do `forward` icon in `NeumorphicAppBarIcons` and delete `Transform.rotate`, then need rename `bool reversedIcon` to `forward`